menuinst/platforms/win_utils/registry.py

```diff
@@ -107,16 +107,10 @@ def unregister_file_extension(extension, identifier, mode="user"):
         with winreg.OpenKey(
             root, rf"Software\Classes\{extension}\OpenWithProgids", 0, winreg.KEY_ALL_ACCESS
         ) as key:
-            try:
-                winreg.QueryValueEx(key, identifier)
-            except FileNotFoundError:
-                log.debug(
-                    "Handler '%s' is not associated with extension '%s'", identifier, extension
-                )
-            else:
-                winreg.DeleteValue(key, identifier)
-    except Exception as exc:
-        log.exception("Could not check key '%s' for deletion", extension, exc_info=exc)
+            winreg.QueryValueEx(key, identifier)  #??? Is this necessary?
+            winreg.DeleteValue(key, identifier)
+    except Exception:
```

Comment:
I think the `FileNotFoundError` exeption should be added here. All other errors can still look like a traceback.